### PR TITLE
fix(vast): Remove incorrect URI and Ad Title validation

### PR DESCRIPTION
**Pull request by amokhtar**
# What
Removes validation on ad title and various URIs that was incorrectly marked as required. These URI fields are not required by the VAST 2.0 spec and therefore should not invalidate a VAST ad. This removes the checks and their tests

### CODE REVIEW

File: vast/tracking_test.go [hunk 1]
```diff
@@ -33,4 +32,4 @@ func TestTrackingWithWhitespace(t *testing.T) {
 	if v.Uri != "http://it-is-just-me.com" {
 		t.Errorf("Expected CDATA to be 'http://it-is-just-me.com' instead of '%s'.", v.Uri)
 	}
-}
+}
```
Comment (tiagozortea) [inline vast/tracking_test.go]: Newline at the end.

File: vast/testdata/companion_with_staticresource.xml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-    id="some-id"
-    width="128"
-    height="256"
-    assetWidth="64"
-    assetHeight="32"
-    expandedHeight="16"
-    expandedWidth="8"
-    apiFramework="Vungle Exchange"
-    adSlotId="ad-slot-id">
+        id="some-id"
```
Comment (garukun) [inline vast/testdata/companion_with_staticresource.xml]: Please revert the indent. U might wanna configure autogormatting for this.

File: vast/testdata/companion_with_staticresource.xml [hunk 3]
```diff
@@ -25,4 +25,4 @@
       <With><AChance>Of Jägermeister</AChance></With>
     </CreativeExtension>
   </CreativeExtensions>
-</Companion>
+</Companion>
```
Comment (garukun) [inline vast/testdata/companion_with_staticresource.xml]: Please revert

**Comment by tiagozortea** [comment]
LGTM

**Pull request merged**